apps/vr-tests/.storybook/preview.js

```diff
@@ -1,50 +1,15 @@
 // @ts-check
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31941&commitId=0ceaa222a3585ec5944fe2f81f4667e3081ba361&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Medium.chromium.png|0| Removed|
|Pivot - Overflow.Root.Wide.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Medium.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Wide.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>